app/models/post.rb

```diff
@@ -0,0 +1,9 @@
+class Post < ActiveRecord::Base
+  has_attached_file :image, styles: { large: "600x600>", medium: "300x300>", thumb: "150x150#" }
```

Comment:
Metrics/LineLength: Line is too long. [96/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)